### PR TITLE
Limit number of parallel jobs based on memory

### PR DESCRIPTION
**Pull request by rvykydal**
https://github.com/rhinstaller/kickstart-tests/issues/524

### CODE REVIEW

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Comment by jstodola** [comment]
Thank you!

File: containers/runner/launch [hunk 1]
```diff
@@ -41,12 +54,14 @@ Options:
                                        rhinstaller/kickstart-tests workflow artifacts.)
  --defaults DEFAULTS_SH_FILE          Path to file with overrides to scripts/defaults.sh
  --run-args ARGUMENTS                 Extra $CRUN options/arguments (space separated)
+ --recommended-jobs                   Get the number of jobs (--jobs option) run in parallel
+                                      by default, based on number of CPUs and RAM detected
```
Comment (jkonecny12) [inline containers/runner/launch:58]: Could you please improve the message to be obvious what this option is doing?

Something like:
``Print number of jobs (see --jobs option) used by default and exit. These jobs are calculated from number of CPU and size of RAM.``

**Review by jkonecny12 — COMMENTED** [review]
Looks good to me otherwise.

File: containers/runner/launch (hunk 1 above)
Comment (rvykydal) [reply containers/runner/launch]: Updated

Comment (rvykydal) [reply containers/runner/launch]: @jkonecny12 - is it ok now?

**Review by jkonecny12 — APPROVED** [review]
Looks great to me now! Thanks a lot!

**Pull request merged**